### ISSUE TITLE
Does performance increase when learning more steps?

### ISSUE THREAD

**Issue by daegonYu** [McGill-NLP/llm2vec#106]
hello!

In the LLM2Vec paper, only 1000 steps were trained when learning MNTP and SimCSE. Will learning more steps result in higher performance? Or is the performance improvement minimal? Also, since it is an English-based model when learning other languages, do you expect it to help performance when learning more steps than 1000 steps? Your answer would be of great help!

**Comment by vaibhavad** [comment]
Hi @daegonYu, as we don't want to overfit to the MTEB benchmark, we only measure the performance on it once. The 1000 step checkpoint was selected by checking performance on single small datasets from MTEB. The performance on this selected validation set did not increase after 1000 steps. 

However, It is theoretically possible that some other step checkpoint gives better performance on entire MTEB evaluation suite.

**Comment by vaibhavad** [comment]
Closing as it is stale. Feel free to re-open if you have any more questions.
